lib/web_ui/lib/src/engine/pointer_binding.dart

```diff
@@ -543,25 +495,23 @@ List<ui.PointerData> _convertWheelEventToPointerData(
       break;
   }
 
-  final List<ui.PointerData> data = <ui.PointerData>[];
-  _ensureMouseDeviceAdded(data, event.client.x, event.client.y, event.buttons,
-      event.timeStamp, _mouseDeviceId);
-  data.add(ui.PointerData(
-    change: ui.PointerChange.hover,
-    timeStamp: _eventTimeStampToDuration(event.timeStamp),
-    kind: ui.PointerDeviceKind.mouse,
-    signalKind: ui.PointerSignalKind.scroll,
-    device: _mouseDeviceId,
-    physicalX: event.client.x * ui.window.devicePixelRatio,
-    physicalY: event.client.y * ui.window.devicePixelRatio,
-    buttons: event.buttons,
-    pressure: 1.0,
-    pressureMin: 0.0,
-    pressureMax: 1.0,
-    scrollDeltaX: deltaX,
-    scrollDeltaY: deltaY,
-  ));
-  return data;
+  return <ui.PointerData>[
+      ui.PointerData(
+      change: ui.PointerChange.hover,
+      timeStamp: _eventTimeStampToDuration(event.timeStamp),
+      kind: ui.PointerDeviceKind.mouse,
+      signalKind: ui.PointerSignalKind.scroll,
+      device: _mouseDeviceId,
+      physicalX: event.client.x * ui.window.devicePixelRatio,
```

Comment:
This looks a bit weird as well, the device pixel ration is also multiplied in framework converter.dart

Reply:
This is correct. For us the "device pixel" is "CSS pixel" which is logical, not physical. So we always report device pixel ratio of 1.0 to the framework and convert it internally. This may change in the future, but for now let's keep it despite the weirdness.